docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -50,49 +50,48 @@ The primary challenge of migrating a .NET Framework app to .NET Core is that its
 
 The Bean Trader sample has the following top-level NuGet dependencies:
 
-* [**Castle.Windsor, version 4.1.1**](https://www.castleproject.org/projects/windsor/)  
-This package targets .NET Standard 1.6, so it works on .NET Core.
+- [**Castle.Windsor, version 4.1.1**](https://www.castleproject.org/projects/windsor/)  
+> This package targets .NET Standard 1.6, so it works on .NET Core.
 
-* [**Microsoft.CodeAnalysis.FxCopAnalyzers, version 2.6.3**](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/2.6.3)  
-This is a meta-package, so it's not immediately obvious which platforms it supports, but [documentation](https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisfxcopanalyzers) indicates that its newest version (2.9.2) will work for both .NET Framework and .NET Core.
+- [**Microsoft.CodeAnalysis.FxCopAnalyzers, version 2.6.3**](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/2.6.3)  
+> This is a meta-package, so it's not immediately obvious which platforms it supports, but [documentation](https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisfxcopanalyzers) indicates that its newest version (2.9.2) will work for both .NET Framework and .NET Core.
 
-* [**Nito.AsyncEx, version 4.0.1**](https://www.nuget.org/packages/Nito.AsyncEx/4.0.1)  
-This package doesn't target .NET Core, but the newer 5.0 version does. This is common when migrating because many NuGet packages have added .NET Standard support recently, but older project versions will only target .NET Framework. If the version difference is only a minor version difference, it's often easy to upgrade to the newer version. Because this is a major version change, you need to be cautious upgrading since there could be breaking changes in the package. There is a path forward, though, which is good.
+- [**Nito.AsyncEx, version 4.0.1**](https://www.nuget.org/packages/Nito.AsyncEx/4.0.1)  
+> This package doesn't target .NET Core, but the newer 5.0 version does. This is common when migrating because many NuGet packages have added .NET Standard support recently, but older project versions will only target .NET Framework. If the version difference is only a minor version difference, it's often easy to upgrade to the newer version. Because this is a major version change, you need to be cautious upgrading since there could be breaking changes in the package. There is a path forward, though, which is good.
```

Comment:
Fixes lint error.

```suggestion

> This package doesn't target .NET Core, but the newer 5.0 version does. This is common when migrating because many NuGet packages have added .NET Standard support recently, but older project versions will only target .NET Framework. If the version difference is only a minor version difference, it's often easy to upgrade to the newer version. Because this is a major version change, you need to be cautious upgrading since there could be breaking changes in the package. There is a path forward, though, which is good.
```